### ISSUE TITLE
Notify people once the foosball table is free

### ISSUE THREAD

**Issue by swehner** [swehner/foos#20]
If it's busy at the moment users should be able to tell the bot to notify them once the table is free again.

**Comment by timomanson** [comment]
Hi, ich habe probleme das ganze auf pi 3 zum laufen zu bringen, vielleicht kannst du mich ein wenig unterstützen....

**Comment by timomanson** [comment]
Hi,

also istalliert habe ich es, jetzt will ich versuchen ein app zu bauen, um es via Bluetooth über ein tab steuern zu können. 
Du hast nicht zufällig sowas schonmal gemacht ?

Ich bin eigentlich flugzeugmechaniker und ich hab lediglich mal nen Linux Grundlagen kurs besucht.

MFG

Gesendet von Mail für Windows 10

Von: swehner
Gesendet: Donnerstag, 10. September 2020 19:06
An: swehner/foos
Cc: timomanson; Comment
Betreff: Re: [swehner/foos] Notify people once the foosball table is free(#20)

Hallo! Natürlich, du kannst mich entweder über email anschreiben, oder öffne einfach ein neues Issue mit dem Problem dass du hast (um nicht hier auf diesem Issue weiter zu schreiben!)

**Comment by swehner** [comment]
Hi Timo,

Was würdest du über das Telefon steuern wollen?
Ich denke, dass einfachst wäre ein Webinterface hinzuzufügen. Anstatt
bluetooth und eine spezielle App, köntest du das ganze über Wifi steuern,
wenn die RaspberryPi und das Telefon im selben Netzwerk sind.
Wäre auf jeden Fall recht einfach und würde auch auf allen Telefonen
funktionieren.
Ich denke es wäre recht einfach das als plugin zu schreiben und einen
kleinen Webserver in Python laufen zu lassen mit einenm einfachen interface.
Was hältst du davon?

**Comment by timomanson** [comment]
Hi, Ich finde es ziemlich cool das du dich überhaupt mit dem Thema befasst. Ich würde da vielleicht noch einen Schritt weiter gehen. Was würdest du davon halten das ganze im Internet kostenlos für Vereine usw. Zu verbreiten, ich glaube das es ziemlich erfolgreich werden könnte, wenn man sich als liega damit vernetzen könnte. Und dann auch noch kostenlos. Die Hardware könnte man mit vorprogrammierter Software komplett für mit nem Aufschlag für die Arbeit vertreiben. Den respberry  fertig verlötete Lichtschranken einen Stecker bauen und wir geben das als swehner Auto foos  bekannt. Was sagst du dazu? Wie aber schon gesagt, ich bin was Apps programmieren betrifft eine echte Niete MfgTimo Methling Von meinem Huawei-Telefon gesendet-------- Ursprüngliche Nachricht --------Von: swehner <notifications@github.com>Datum: So., 13. Sep. 2020, 16:23An: swehner/foos <foos@noreply.github.com>Cc: timomanson <timo-methling@gmx.de>, Comment <comment@noreply.github.com>Betreff: Re: [swehner/foos] Notify people once the foosball table is free (#20)

Hi Timo,



Was würdest du über das Telefon steuern wollen?

Ich denke, dass einfachst wäre ein Webinterface hinzuzufügen. Anstatt

bluetooth und eine spezielle App, köntest du das ganze über Wifi steuern,

wenn die RaspberryPi und das Telefon im selben Netzwerk sind.

Wäre auf jeden Fall recht einfach und würde auch auf allen Telefonen

funktionieren.

Ich denke es wäre recht einfach das als plugin zu schreiben und einen

kleinen Webserver in Python laufen zu lassen mit einenm einfachen interface.

Was hältst du davon?

**Comment by timomanson** [comment]
Hi, Ich habe die Version ohne arduino gebaut. Wie werden denn die Schalter angeschlossen? Oder sind das jeweils yellow + - und black+-? Ich versteh das nicht ganz... Mfg Von meinem Huawei-Telefon gesendet-------- Ursprüngliche Nachricht --------Von: swehner <notifications@github.com>Datum: So., 13. Sep. 2020, 16:23An: swehner/foos <foos@noreply.github.com>Cc: timomanson <timo-methling@gmx.de>, Comment <comment@noreply.github.com>Betreff: Re: [swehner/foos] Notify people once the foosball table is free (#20)

Hi Timo,



Was würdest du über das Telefon steuern wollen?

Ich denke, dass einfachst wäre ein Webinterface hinzuzufügen. Anstatt

bluetooth und eine spezielle App, köntest du das ganze über Wifi steuern,

wenn die RaspberryPi und das Telefon im selben Netzwerk sind.

Wäre auf jeden Fall recht einfach und würde auch auf allen Telefonen

funktionieren.

Ich denke es wäre recht einfach das als plugin zu schreiben und einen

kleinen Webserver in Python laufen zu lassen mit einenm einfachen interface.

Was hältst du davon?

**Comment by swehner** [comment]
Also die Tasten sind für + und - (hoch/runter) jeweils für die beiden teams
(schwarz und gelb). Hier ist ein schema:
https://github.com/swehner/foos/blob/master/doc/foos_RPi_only_schema.png
dort siehst du wie sie angeschlossen werden. Hier hast du den pinout:
https://www.raspberrypi.org/documentation/usage/gpio/
Hoffentlich hilfts!

**Comment by timomanson** [comment]
Hi,

wäre es machbar die tasten mit einem Touchscreen zu generieren ?

MFG

Timo

Gesendet von Mail für Windows 10

Von: swehner
Gesendet: Dienstag, 29. September 2020 23:00
An: swehner/foos
Cc: timomanson; Comment
Betreff: Re: [swehner/foos] Notify people once the foosball table is free(#20)

Also die Tasten sind für + und - (hoch/runter) jeweils für die beiden teams
(schwarz und gelb). Hier ist ein schema:
https://github.com/swehner/foos/blob/master/doc/foos_RPi_only_schema.png
dort siehst du wie sie angeschlossen werden. Hier hast du den pinout:
https://www.raspberrypi.org/documentation/usage/gpio/
Hoffentlich hilfts!

**Comment by timomanson** [comment]
Hi, mal ne Dumme Frage, und auf dem Bild sind die ir receiver die ich habe. Ist out = Data? MfgTimo Von meinem Huawei-Telefon gesendet-------- Ursprüngliche Nachricht --------Von: swehner <notifications@github.com>Datum: Di., 29. Sep. 2020, 23:00An: swehner/foos <foos@noreply.github.com>Cc: timomanson <timo-methling@gmx.de>, Comment <comment@noreply.github.com>Betreff: Re: [swehner/foos] Notify people once the foosball table is free (#20)

Also die Tasten sind für + und - (hoch/runter) jeweils für die beiden teams

(schwarz und gelb). Hier ist ein schema:

https://github.com/swehner/foos/blob/master/doc/foos_RPi_only_schema.png

dort siehst du wie sie angeschlossen werden. Hier hast du den pinout:

https://www.raspberrypi.org/documentation/usage/gpio/

Hoffentlich hilfts!



On Mon, Sep 28, 2020 at 11:31 PM timomanson <notifications@github.com>

wrote:

**Comment by timomanson** [comment]
Hi, 
ich habe ein problem
-------------------------------

* Library deps:
[ OK] libtiff.so found
[ OK] libf77blas.so found
[ OK] libjpeg.so found
### **[ERR] libpng.so not found!**
====> Try: sudo apt-get install libpng-dev 

------------------------------------------------------------------------------------------------------
[ERR] libpng.so not found!

bkomme ich auch nicht mit 

sudo apt-get install libpng-dev 

installiert , was kann man da machen ?

MFG

TIMO

**Comment by swehner** [comment]
Hi Timo,

Was hat du denn auf der RapsberryPi installiert Raspbian? Welche version?
Wahrscheinlich heisst die lib anders, kannst du mal "dpkg -S libpng" laufen
lassen?

**Comment by timomanson** [comment]
Ja, aber erst am Wochenende. Bin im Ausland. Trotzdem danke schon mal. Ich melde mich dann Von meinem Huawei-Telefon gesendet-------- Ursprüngliche Nachricht --------Von: swehner <notifications@github.com>Datum: Di., 13. Okt. 2020, 22:30An: swehner/foos <foos@noreply.github.com>Cc: timomanson <timo-methling@gmx.de>, Comment <comment@noreply.github.com>Betreff: Re: [swehner/foos] Notify people once the foosball table is free (#20)

Hi Timo,



Was hat du denn auf der RapsberryPi installiert Raspbian? Welche version?

Wahrscheinlich heisst die lib anders, kannst du mal "dpkg -S libpng" laufen

lassen?

**Comment by timomanson** [comment]
Hi, Hab mal " dpkg -S libpng" laufen lassen, das kam dabei heraus Ich kann da nicht viel mit anfangen.. MfgTimo Von meinem Huawei-Telefon gesendet-------- Ursprüngliche Nachricht --------Von: swehner <notifications@github.com>Datum: Di., 13. Okt. 2020, 22:30An: swehner/foos <foos@noreply.github.com>Cc: timomanson <timo-methling@gmx.de>, Comment <comment@noreply.github.com>Betreff: Re: [swehner/foos] Notify people once the foosball table is free (#20)

Hi Timo,



Was hat du denn auf der RapsberryPi installiert Raspbian? Welche version?

Wahrscheinlich heisst die lib anders, kannst du mal "dpkg -S libpng" laufen

lassen?
